app/views/campaign_requests/_request_details.html.erb

```diff
@@ -2,29 +2,153 @@
-    <%= r.input :success_measure, as: :text, label: "How will you measure success?", required: true, input_html: {:class => "input-md-6", :rows => 6, :cols => 50, :"aria-required" => true } %>
+    <%= r.input :proposed_url,
+                label: "Proposed URL (in the form of xxxxx.campaign.gov.uk or xxxxx.gov.uk)",
+                required: true, input_html: { class: "input-md-6",
+                                              :"aria-describedby" => "urlHelpBlock",
+                                              :"aria-required" => true } %>
+    <p class="help-block">
+      If applying for Campaigns Platform it is in the form of xxxxx.campaign.gov.uk, or otherwise
+      if a microsite, standard format is  in the form of xxxxxx.gov.uk.
+    </p>
 
-    <%= r.input :proposed_url, label: "Proposed URL (in the form of xxxxx.campaign.gov.uk)", required: true, input_html: { class: "input-md-6", :"aria-describedby" => "urlHelpBlock", :"aria-required" => true } %>
+    <%= r.input :site_metadescription,
+                as: :text,
+                label: "Site metadescription (appears in search results)",
+                required: true,
+                input_html: {:class => "input-md-6",
+                             :rows => 6,
+                             :cols => 50,
+                             :"aria-required" => true,
+                             :"aria-describedby" => "seoHelpBlock" } %>
+    <p class="help-block">
+      The meta description is a short, clear sentence how you want the site described as it appears
+      in Google search
+    </p>
 
-    <%= r.input :site_metadescription, as: :text, label: "Site metadescription (appears in search results)", required: true, input_html: {:class => "input-md-6", :rows => 6, :cols => 50, :"aria-required" => true, :"aria-describedby" => "seoHelpBlock" } %>
+    <%= r.input :cost_of_campaign, label: "Site build budget / costs (and overall campaign cost, if applicable)", required: true, input_html: { class: "input-md-6" } %>
+    <p class="help-block">
+      If you are applying for the Campaign Platform page, this is free of any build charge
+    </p>
 
-    <%= r.input :cost_of_campaign, label: "Cost of campaign", required: true, input_html: { class: "input-md-6" } %>
+    <%= r.input :ga_contact_email, label: "Contact details for Google Analytics leads (Gmail accounts only)", required: true, input_html: {:class => "input-md-6", :rows => 6, :cols => 50, :"aria-required" => true } %>
+    <p class="help-block">
+      GCS centralise and coordinate access to Google Analytics, although departments will themselves
+      be responsible for analysing their site’s performance.
+    </p>
+    <p class="help-block">
+      If you need to set up a bespoke analytics package independent of centralised/GCS GA for the purposes
+      of your marketing, please ​provide the following with access ​webmaster@cabinetoffice.gov.uk &
```

Comment:
Github has flagged this `&` in red.  Perhaps we should change it to either `and` or `&amp;`.

Reply:
Changed to "and"